ti/instrumentation/java/maven.go

```diff
@@ -36,6 +36,26 @@ func (m *mavenRunner) AutoDetectPackages(workspace string) ([]string, error) {
 	return DetectPkgs(workspace, m.log, m.fs)
 }
 
+func (m *mavenRunner) AutoDetectTests(ctx context.Context, workspace string) ([]ti.RunnableTest, error) {
```

Comment:
Please add docstring with comments about the assumptions.

Reply:
Done